lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -461,10 +461,11 @@ def add_emv_metadata(post, creditcard)
         post[:metadata][:card_read_method] = creditcard.read_method if creditcard.respond_to?(:read_method)
       end
 
-      def fetch_application_fees(identification, options = {})
+      def fetch_application_fee(identification, options = {})
         options.merge!(:key => @fee_refund_api_key)
 
-        commit(:get, "application_fees?charge=#{identification}", nil, options)
+        fetch_charge = commit(:get, "charges/#{identification}", nil, options)
```

Comment:
Do you think we need `#{CGI.escape(identification)}` here? I'm not sure what values identification could hold, but looks like we've escaped this variable in other calls.

Reply:
👍 Good call.